### PR TITLE
Migrate readme yaml

### PR DESCRIPTION
**Pull request by vadim-gleyf**
## what
* Add `README.yaml`

## why
* Standardize README

### CODE REVIEW

**Comment by vadim-gleyf** [comment]
depends on https://github.com/cloudposse/build-harness/issues/72

File: README.yaml [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  -----
+  ![atom-image-assistant](https://cloud.githubusercontent.com/assets/118112/24306827/2db2494a-107f-11e7-969a-2581851aa816.gif)
+
+# Contributors to this project
+contributors:
+  - name: "Igor Rodionov"
+    github: "goruha"
+  - name: "Tamas Nagy"
+    github: "tlnagy"
+  - name: "Rob \\| Labs"
```
Comment (osterman) [inline README.yaml:122]: Remvoe the `\\|` =)

File: README.yaml [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  * Per-file asset directory
+  ```
+  ![](README.assets/README-684b45a8.jpg)
+  ```
+
+  * Preserve Original filename
+  ```
+  ![](README.assets/README-Fort-*ssHole.jpg)
+  ```
+
```
Comment (osterman) [inline README.yaml:107]: Add a `references` section with a link to the original repo

File: README.yaml [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+# This is the canonical configuration for the `README.md`
+# Run `make readme` to rebuild the `README.md`
+#
+
+# Name of this project
+name: atom-markdown-image-assistant
+
+# Logo for this project
+#logo: docs/logo.png
+
```
Comment (osterman) [inline README.yaml:12]: Add a `copyrights` section with attribution for original author

**Review by osterman — CHANGES_REQUESTED** [review]
See inline comments

File: README.yaml (hunk 1 above)
Comment (vadim-gleyf) [reply README.yaml:122]: This character breaks down the table markdown.

**Pull request closed without merging**